### DISCUSSION TITLE
is getStaticProps deprecated in app router in next.js v15 ?

### DISCUSSION THREAD

**Discussion by nicevlhjfihtqngkldl** [vercel/next.js#69275] [Help]
### Summary

Hi
I cannot find documents about getStaticProps in app router in next.js v15 ?
Is it deprecated?
Thanks for help!

**Comment by ZoranRavic** [comment]
`getStaticProps` is only for `pages` router.

You don't actually need this if you are using the `app` router, since server components are rendered at build time unless they use dynamic values like headers or cookies.
